packages/tools/generator-widget/README.md

```diff
@@ -15,7 +15,13 @@ This generator uses the Yeoman scaffolding tool to let you quickly create a [Men
 
 ## Installation
 
-1. Install [node.js](https://nodejs.org/) (version >= 10.15).
+1. Install [node.js](https://nodejs.org/) (version >= 12).
+1. Install [NPM](https://www.npmjs.com/package/npm) (version < 7):
+
+    ```bash
+     npm install -g npm@"<7"
+    ```
+
```

Comment:
Lets add a warning for users already with npm 7 to use `--legacy-peer-deps` combined with npm install

Reply:
Same comment needs to be added in piw-tools README

Reply:
Changed it